### ISSUE TITLE
Slack → Mattermost custom avatars not synced (excepted for mine)

### ISSUE THREAD

**Issue by ArchangeGabriel** [42wim/matterbridge#472]
**Describe the bug**
This is a follow up of #454. Basically, users sending messages from Slack to Mattermost fall into one of the three following categories:
1. They use a default avatar from Slack, in which case this one is properly displayed on mattermost side.
2. Me, using a custom avatar on Slack which is properly displayed on Mattermost.
3. Anyone else using a custom avatar, which is replaced by the default bridge logo.

**To Reproduce**
Have anyone excepted myself post on the Slack side.

**Expected behavior**
Custom avatars should be displayed properly on Mattermost side, not only mine.

**Screenshots/debug logs**
Logs: see https://github.com/42wim/matterbridge/issues/454#issuecomment-402680561, https://github.com/42wim/matterbridge/issues/454#issuecomment-402683942 and https://github.com/42wim/matterbridge/issues/454#issuecomment-402685913.

**Environment (please complete the following information):**
 - OS: Arch Linux
 - Matterbridge version: 1.11.0 + https://github.com/42wim/matterbridge/commit/fa0c4025f71373310f08f6b85ae38fe58876e6f8

**Additional context**
Config: https://github.com/42wim/matterbridge/issues/454#issuecomment-402850611

As well as a new addition to gateways:
```
[[gateway]]
name="gateway_test"
enable=true

    [[gateway.inout]]
    account="mattermost.<redacted>"
    channel="test"

    [[gateway.inout]]
    account="slack.<redacted>"
    channel="test"
```

Tests have been made in both the `samechannel` gateway and the `gateway_test`.

**Comment by 42wim** [comment]
This should be fixed in master now, try a dev binary from https://bintray.com/42wim/nightly/Matterbridge/_latestVersion or rebuild from source.

If still an issue, please reopen!

**Comment by ArchangeGabriel** [comment]
That commit fixed, thanks! You’re doing a stellar job. :)

I’m ready to update to 1.11.1 too, just missing the PGP signature on the tarball. ;)

**Comment by 42wim** [comment]
Thanks ;)
The signature should be there now

**Comment by ArchangeGabriel** [comment]
On a slide note, thanks to you I’ve figured out why it worked for me and not the others, and this helped me fix another issue: under Mattermost my Slack username was shown, while other people have their normal name. But that’s because they have a DisplayName while I was only having a FullName. Added a display name for myself and now everything is OK.

Since you’ve also fixed #452 in that release, this is now looking very pretty.

#191 would make this awesome, #474 (and the reverse way) would be the cherry on the top!

Keep up the great work! :D
